flink-connector-aws/flink-connector-dynamodb/pom.xml

```diff
@@ -76,6 +77,16 @@ under the License.
             <artifactId>dynamodb-enhanced</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.datatype</groupId>
+            <artifactId>jackson-datatype-jsr310</artifactId>
+        </dependency>
+
```

Comment:
Let's remove these libraries too if we are removing the `RecordObjectMapper`

Reply:
removed